metecho/api/models.py

```diff
@@ -79,6 +79,11 @@ class User(HashIdMixin, AbstractUser):
     devhub_username = StringField(blank=True, default="")
     allow_devhub_override = models.BooleanField(default=False)
     agreed_to_tos_at = models.DateTimeField(null=True, blank=True)
+    onboarded_at = models.DateTimeField(
```

Comment:
@jgerigmeyer do we want to know which onboarding flows a user has completed?  My concern with using a timestamp would be how to know whether to guide someone through the other tours if they've just completed one.

Reply:
No, we don't want to force users to complete all of the walkthroughs. We just show them the modal once initially, then that's it.